content/doc/book/installing/index.adoc

```diff
@@ -220,33 +242,128 @@ Here, "8081" was chosen but you can put another port available.
+
+[source,bash]
+----
+sudo systemctl start jenkins.service
 ----
 
-Then, you can install Jenkins.  The following command also ensures you have java installed.
+You can check the status of the Jenkins service using the command:
 
 [source,bash]
 ----
-sudo dnf upgrade && sudo dnf install jenkins java
+sudo systemctl status jenkins.service
 ----
 
-Next, start the Jenkins service.
+If everything has been set up correctly, you should see an output like this:
+
+[source,bash]
+----
+Loaded: loaded (/etc/rc.d/init.d/jenkins; generated)
+Active: active (running) since Tue 2018-11-13 16:19:01 +03; 4min 57s ago
+----
+
+[NOTE]
+====
+If you have a firewall installed, you must add Jenkins as an exception.
+You must change `YOURPORT` in the script below to the port you want to use.
+Port `8080` is the most common.
+
+[source,bash]
+----
+YOURPORT=8080
+PERM="--permanent"
+SERV="$PERM --service=jenkins"
+
+firewall-cmd $PERM --new-service=jenkins
+firewall-cmd $SERV --set-short="Jenkins ports"
+firewall-cmd $SERV --set-description="Jenkins port exceptions"
+firewall-cmd $SERV --add-port=$YOURPORT/tcp
+firewall-cmd $PERM --add-service=jenkins
+firewall-cmd --zone=public --add-service=http --permanent
+firewall-cmd --reload
+----
+
+====
+
+==== Red Hat / CentOS
+
+You can install Jenkins through `yum` on Red Hat Enterprise Linux, CentOS, and other Red Hat based distributions.
+You need to choose either the Jenkins Long Term Support release or the Jenkins weekly release.
+
+===== Long Term Support release
+
+A link:/download/lts/[LTS (Long-Term Support) release] is chosen every 12 weeks from the stream of regular releases as the stable release for that time period.
+It can be installed from the link:https://pkg.jenkins.io/redhat-stable/[`redhat-stable`] yum repository.
+
+[source,bash]
+----
+sudo wget -O /etc/yum.repos.d/jenkins.repo \
+    https://pkg.jenkins.io/redhat-stable/jenkins.repo
+sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
+sudo yum upgrade
+sudo yum install jenkins java-1.8.0-openjdk-devel
+----
+
+===== Weekly release
+
+A new release is produced weekly to deliver bug fixes and features to users and plugin developers.
+It can be installed from the link:https://pkg.jenkins.io/redhat/[`redhat`] yum repository.
+
+[source,bash]
+----
+sudo wget -O /etc/yum.repos.d/jenkins.repo \
+    https://pkg.jenkins.io/redhat/jenkins.repo
+sudo rpm --import https://pkg.jenkins.io/redhat/jenkins.io.key
+sudo yum upgrade
+sudo yum install jenkins java-1.8.0-openjdk-devel
+----
+
+===== Start Jenkins
+
+You can start the Jenkins service with the command:
 
 [source,bash]
 ----
-sudo service jenkins start
-sudo chkconfig jenkins on
+sudo systemctl start jenkins.service
 ----
 
-You can check the status of the Jenkins service using this systemctl command:
+You can check the status of the Jenkins service using the command:
 
 [source,bash]
 ----
-systemctl status jenkins
+sudo systemctl status jenkins.service
```

Comment:
Does the .service need adding?

Reply:
I took  the command from the [Red Hat Enterprise Linux 7 Administrators Guide](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/sect-Managing_Services_with_systemd-Services#tabl-Managing_Services_with_systemd-Services-systemctl).  I believe it works without the `.service` suffix, but I assume they placed it in that guide intentionally.

Reply:
As a counterpoint that may indicate the Administrators Guide is incorrect, the [Fedora Project Guide](https://docs.fedoraproject.org/en-US/quick-docs/understanding-and-administering-systemd/#starting-stopping-and-querying-systemd-services) uses the command without the `.service` suffix.

Debian also does not use the `.service` suffix in [their examples](https://wiki.debian.org/systemd#Managing_services).

I think I'll drop the `.service` suffix and assume that Red Hat Enterprise Linux 7 won't break (since I've previously used the command without the suffix on CentOS 7).

Reply:
See 9df98c2119918c852721c77a3566fe258ec6cc87 for the change.  Thanks for the input!